vignettes/package_structure.Rmd

```diff
@@ -128,7 +128,6 @@ This file implements the `P`, `Q` and `R` matrix calculations which are needed f
 ### `Makevars`
 
 This file specifies additional flags used in the source code compilation. 
-We specify here the use of `C++14` such that `std::make_unique()` is available.
 Also we ask `TMB` to use the `TMB` automatic differentiation framework (instead of the default `CppAD` framework) to be more efficient, and disable useless warnings from the `Eigen` library compilation.
```

Comment:
```suggestion
W ask `TMB` to use the `TMB` automatic differentiation framework (instead of the default `CppAD` framework) to be more efficient, and disable useless warnings from the `Eigen` library compilation.
```